crates/cxx-qt-gen/src/generator/naming/namespace.rs

```diff
@@ -15,7 +15,10 @@ pub struct NamespaceName {
 
 impl From<&ParsedQObject> for NamespaceName {
     fn from(qobject: &ParsedQObject) -> Self {
-        NamespaceName::from_pair_str(&qobject.namespace, &qobject.qobject_ty.ident_left)
+        NamespaceName::from_pair_str(
```

Comment:
why is `NamespaceName` called `from_pair_str` (when it's a string and an ident ?) whereas `QObjectName` is called `from_name_and_ident` ?

Reply:
No idea where this came from :shrug: 
Renamed to `from_namespace_and_ident`

Reply:
:sweat_smile: